sky/packages/sky/lib/src/widgets/event_recorder.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+
+enum EventRecorderMode {
+  stop,
+  record
+}
+
+typedef void EventsReady(List<PointerInputEvent> events);
```

Comment:
We usually use the suffix "Callback" for these types.  Not sure if having that suffix is all that valuable, but there's some value in being consistent.